.eslintrc

```diff
@@ -136,19 +134,12 @@
     "react/react-in-jsx-scope": "error",
     "react/self-closing-comp": "error",
     "react/sort-prop-types": "error",
-    "rest-spread-spacing": ["error", "never"],
-    "semi": ["error", "never"],
-    "space-before-blocks": "error",
-    "space-before-function-paren": ["error", { "anonymous": "always", "named": "never" }],
-    "space-in-parens": "error",
-    "space-infix-ops": "error",
-    "space-unary-ops": ["error", { "words": true, "nonwords": false }],
-    "spaced-comment": ["error", "always", { "exceptions": ["-"]}],
-    "template-curly-spacing": "error",
-    "template-tag-spacing": ["error", "never"],
-    "unicode-bom": ["error", "never"],
```

Comment:
Yes. Prettier will take care of all the formatting (as it will reformat everything), so these rules are no longer needed.

https://github.com/prettier/eslint-config-prettier disables all those rules, but as its loaded using `extends` all the rules in `.eslintrc` are overriding it. So I went over all the rules and removed the ones specified in `eslint-config-prettier` as useless when using Prettier.

Reply:
@ianstormtaylor Because eslint is conflicting with some styling rules.  I remember eslint-config-prettier has a command to check conflicting rules